app/views/judgements/_point_requests.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="uk-overflow-container">
+  <table class="uk-table uk-table-striped uk-table-hover uk-text-center uk-text-nowrap">
```

Comment:
This table contains very few data compared to its size, and even with a relatively smaller group, it might be difficult to follow which point belongs to which principle. (https://imgur.com/a/jvJJ1KQ) Maybe adding vertical lines between the columns would help with this.